Menu/Menu.js

```diff
@@ -1,10 +1,12 @@
 
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+ menu.classList.toggle('menu--open')
+ // Toggle the "menu--open" class on your menu refence. 
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector('.menu')
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelector('.menu-button')
 // Using your menuButton reference, add a click handler that calls toggleMenu
+menuButton.addEventListener('click', toggleMenu)
```

Comment:
Well done just referencing the function. Since this is not a class component you don't need to call the method inside an anonymous arrow function because there is no this keyword to bind.